File: frontend/src/models/HardwareStore.ts
export type HardwareStoreItem = {
  id: number;
  name: string;
  price: number;
  description: string;
  quantity: number;
  category: string;
};

export type CreateHardwareStoreItem = {
  name: string;
  price: number;
  description: string;
  quantity: number;
  category: string;
};

File: backend/backend/src/main/java/edu/bbte/idde/abim2109/backend/model/HardwareStore.java
package edu.bbte.idde.abim2109.backend.model;

import lombok.*;

@Data
@EqualsAndHashCode(callSuper = true)
@ToString(callSuper = true)
@NoArgsConstructor
@AllArgsConstructor
public class HardwareStore extends BaseEntity {
    private String name;
    private String category;
    private String description;
    private Integer price;
    private Integer quantity;
}